libs/pages/services/src/lib/ui/page-job-create/step-summary/step-summary.tsx

```diff
@@ -140,6 +143,30 @@ export function StepSummary(props: StepSummaryProps) {
             </ul>
           </Section>
 
+          {(props.dockerfileData?.dockerfile_path || props.dockerfileData?.dockerfile_raw) && (
+            <Section className="rounded border border-neutral-250 bg-neutral-100 p-4">
+              <div className="flex justify-between">
+                <Heading>Dockerfile</Heading>
+                <Button type="button" variant="plain" size="md" onClick={props.gotoDockerfileJob}>
+                  <Icon className="text-base" iconName="gear-complex" />
+                </Button>
+              </div>
+              <ul className="flex list-none flex-col gap-2 text-sm text-neutral-400">
+                {props.dockerfileData.dockerfile_path && (
+                  <li>
+                    <strong className="font-medium">Dockerfile path:</strong> {props.dockerfileData.dockerfile_path}
+                  </li>
+                )}
+                {props.dockerfileData.dockerfile_raw && (
+                  <li>
+                    <strong className="font-medium">From raw Dockerfile:</strong> {}
+                    {truncateText(props.dockerfileData.dockerfile_raw, 50)}
```

Comment:
Missing "..." at the end of the truncate maybe
```suggestion
                    {truncateText(props.dockerfileData.dockerfile_raw, 50)}...
```

Reply:
Fixed
https://github.com/Qovery/console/pull/1433/commits/ac99ffdd49c46ffaa5d58b45f33315f67a19329d